charts/uplink/templates/tests/secret_test.yaml

```diff
@@ -0,0 +1,40 @@
+suite: test secret.yaml
+templates:
+  - secret.yaml
+tests:
+  - it: should ensure the secret has correct metadata
+    set:
+      postgresql.url: "example-url"
+      client.id: "example-id"
+      client.secret: "example-secret"
+      sentry.dsn: "example-dsn"
```

Comment:
```suggestion
```

The problem described in #664 is caused when the chart is installed without any additional values other than the defaults (e.g. what is in the `charts/uplink/values.yaml`).

It means that the unit test, in order to simulate this behavior, must have a default nominal case with no values.

You might want  at looking on other chart's tests (ex. https://github.com/jenkins-infra/helm-charts/tree/main/charts/rsyncd/tests) , we tend to have 2 unit test suites:

- `defaults_test.yaml` which simulates a case when calling the chart only with default values
- `custom_values_test.yaml` which simulates a case when passing a set of custom values to verify how they are used